src/components/Checkbox/Checkbox.tsx

```diff
@@ -130,31 +137,43 @@ const CheckboxComponent: ForwardRefRenderFunction<HTMLInputElement, CheckboxProp
                                   !isCheckedOrMixed(state) &&
                                       'tw-border-black-80 tw-bg-white hover:tw-border-black dark:tw-border-white dark:tw-bg-black dark:hover:tw-border-black-20 dark:hover:tw-bg-black-90 group-hover:tw-bg-white group-hover:tw-border-black dark:group-hover:tw-border-black-20 dark:group-hover:tw-bg-black-90',
                                   isCheckedOrMixed(state) &&
+                                      emphasis === CheckboxEmphasis.Weak &&
+                                      'tw-bg-box-neutral-strong tw-text-white hover:tw-bg-box-neutral-inverse dark:tw-text-black group-hover:tw-text-white group-hover:tw-bg-box-neutral-strong dark:group-hover:tw-text-black',
```

Comment:
We are mixing Alias tokens and the Brand tokens here... If I'm not wrong, when we use Alias tokens we should get rid of those styles for `dark:...`. Idk which Alias token would replace the `tw-text-white dark:tw-text-black`, maybe @thisguyscode (Jamie)?

Reply:
`tw-text-white` -> `tw-bg-box-neutral-strong-inverse`

Don't need any `dark:...` stuff